database.py

```
@@ -0,0 +1,16 @@
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from settings import settings
+
+SQLALCHEMY_DATABASE_URL = settings.DATABASE_URL
+
```

Robustness: `settings.DATABASE_URL` is typed as `str | None` in settings. If it ever becomes `None` (misconfigured env), `create_async_engine` will raise a less-helpful error. Consider validating `SQLALCHEMY_DATABASE_URL` and raising a clear error or assert early, e.g.:

```python
if not SQLALCHEMY_DATABASE_URL:
    raise RuntimeError("DATABASE_URL is not configured")
```

This will fail fast with a clearer message.